CONTRIBUTING.md

```diff
@@ -0,0 +1,502 @@
+`module-dir/README.md`, a more detailed list of the module and individual
+tutorial contents is given.  See the Hartree--Fock module README
+[here](https://github.com/psi4/psi4numpy/blob/master/Tutorials/03_Hartree-Fock/README.md)
+for an example.
+
+## Possible Development Workflow
+
+This section outlines a possible workflow for contributing to Psi4NumPy,
+comprised of the following stages:
+
+1. [Creating your own fork of the Psi4NumPy repository](#creating-your-own-fork-of-the-psi4numpy-repository),
+2. [Creating a new feature branch](#creating-a-new-feature-branch) for your content,
+3. [Creating and committing your content](#creating-and-committing-your-content),
+4. [Opening a pull request](#opening-a-pull-request) to merge your content into the parent repository,
+5. [Testing and continuing integration](#testing-and-continuous-integration) of your content,
+6. [Code review](#code-review) with the Psi4NumPy developer community, and
+7. [Merging your content](#merging-your-content) into the main Psi4NumPy repository.
+
+For more information, the suggested workflow for contributing to the Psi4
+repository
+[here](http://psicode.org/psi4manual/1.1/build_obtaining.html#what-is-the-suggested-github-workflow)
+and the GitHub guide [here](https://guides.github.com/introduction/flow/)
+provide graphical representation which visualizes this process.
+
+Note: For clarity when demonstrating Git commands, we will indicate the
+(current-branch) and relative filepath to the `top-level-psi4numpy-dir` within
+the shell prompt.
+
+#### Creating your own fork of the Psi4NumPy repository
+
+To create a fork of the Psi4NumPy repository within your GitHub account, follow
+along with the steps below.  This guide was adapted for our repo from the
+GitHub user documentation
+[here](https://help.github.com/articles/fork-a-repo/).
+
+1. On the main [Psi4NumPy GitHub](https://github.com/psi4/psi4numpy) page, hit the [Fork](https://help.github.com/articles/fork-a-repo/) button to create a fork of the repository on your GitHub account.
+2. From a terminal on your local computer (replacing `johndoe` with your GitHub username):
+```
+# Clone fork `johndoe/psi4numpy` to local
+~$ git clone https://github.com/johndoe/psi4numpy.git
+
+# Navigate into your top-level-psi4numpy-dir
+~$ cd psi4numpy
+
+# Add a connection between your forked repository and the parent repository
+~psi4numpy (master)$ git remote add upstream https://github.com/psi4/psi4numpy.git
+
+# Verify origin & upstream
+~psi4numpy (master)$ git remote -v
+origin    https://github.com/johndoe/psi4numpy.git (fetch)
+origin    https://github.com/johndoe/psi4numpy.git (push)
+upstream    https://github.com/psi4/psi4numpy.git (fetch)
+upstream    https://github.com/psi4/psi4numpy.git (push)
+```
+
+If you have already cloned locally from the `psi4/psi4numpy` master branch:
+1. Create a new fork on your GitHub account according to (1) above
+2. Verify location of `origin`:
```

Comment:
I don't know if its worth having explicit instructions for the one-time case of cloned-upstream-not-fork. It's not like psi4 where there's a semi-painful compile step involved. And in any case, use `rename origin upstream`, not `set-url origin https...`.